src/components/Account/Account.tsx

```diff
@@ -77,7 +77,12 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
     const formErrors: FormErrors = {};
     // Some errors are combined in a single CSV string instead of one string per error
     errors
-      ?.flatMap((e) => e.split(','))
+      ?.flatMap((e) => {
```

Comment:
I think this is an artifact from merging and branching. It shouldn't be necessary anymore or is that not true?

Reply:
ahh yes, you are right

Reply:
OK, done :)